src/header/header.scss

```diff
@@ -274,61 +227,72 @@
 
   .iui-header-button-icon {
```

Comment:
whats the difference between this and `iui-button-icon` ?

Reply:
`.iui-header-button-icon` is the icon that appears within our breadcrumb buttons.  `.iui-button-icon` applies to all of the icons within buttons in the header (breadcrumb button icons as well as notification, help, etc.).